src/routes/shows/+page.svelte

```diff
@@ -1,14 +1,28 @@
 <script lang="ts">
+	import { getShowsByVideoCount } from '$lib/data'
 	import type { PageData } from './$types'
 
 	export let data: PageData
+
+	let sorting: 'alphabetical' | 'most-videos' = 'alphabetical'
```

Comment:
I think it'll feel more TypeScript-y if we make the sorting types an `enum` 🤓

Reply:
[This article](https://dev.to/ivanzm123/dont-use-enums-in-typescript-they-are-very-dangerous-57bh) made me decide to stop using enums entirely, but I'm happy to comply.